tests/network/link_state.go

```diff
@@ -0,0 +1,287 @@
+				libnet.NewBridgeNetAttachDef(bridgeInterfaceNAD3, "br03"))
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		BeforeEach(func() {
+			var ifaceDefault *v1.Interface
+			var iface2 *v1.Interface
+
+			var err error
+			mac1, err = libnet.GenerateRandomMac()
+			Expect(err).NotTo(HaveOccurred())
+			mac2, err = libnet.GenerateRandomMac()
+			Expect(err).NotTo(HaveOccurred())
+			mac3, err = libnet.GenerateRandomMac()
+			Expect(err).NotTo(HaveOccurred())
+
+			ifaceDefault = libvmi.InterfaceWithMac(
+				pointer.P(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				mac1.String())
+			iface2 = libvmi.InterfaceWithMac(libvmi.InterfaceWithState(
+				pointer.P(libvmi.InterfaceDeviceWithBridgeBinding(bridgeInterfaceName2)),
+				v1.InterfaceStateLinkDown),
+				mac2.String())
+
+			vmi := libvmifact.NewFedora(
+				libvmi.WithInterface(*ifaceDefault),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()),
+				libvmi.WithInterface(*iface2),
+				libvmi.WithNetwork(libvmi.MultusNetwork(bridgeInterfaceName2, bridgeInterfaceNAD2)),
+			)
+
+			vm := libvmi.NewVirtualMachine(vmi, libvmi.WithRunStrategy(v1.RunStrategyAlways))
+			vm, err = kubevirt.Client().VirtualMachine(testNamespace).Create(context.Background(), vm, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Eventually(matcher.ThisVM(vm)).WithTimeout(3 * time.Minute).WithPolling(3 * time.Second).Should(matcher.HaveConditionTrue(v1.VirtualMachineInstanceAgentConnected))
+			vmName = types.NamespacedName{Name: vm.GetName(), Namespace: vm.GetNamespace()}
+		})
+
+		It("status and guest should show correct iface state", func() {
+			vmi := waitForStatusToShowExpectedStates(vmName, map[string]v1.InterfaceState{
+				defaultInterfaceName: v1.InterfaceStateLinkUp,
+				bridgeInterfaceName2: v1.InterfaceStateLinkDown,
+			})
+			Expect(console.LoginToFedora(vmi)).To(Succeed())
+			timeout := 5 * time.Second
+			Expect(console.RunCommand(vmi, assertLinkStateCmd(mac1.String(), v1.InterfaceStateLinkUp), timeout)).To(Succeed())
+			Expect(console.RunCommand(vmi, assertLinkStateCmd(mac2.String(), v1.InterfaceStateLinkDown), timeout)).To(Succeed())
+
+			By("flipping the state of both interfaces")
+
+			vm, err := kubevirt.Client().VirtualMachine(vmName.Namespace).Get(context.Background(), vmName.Name, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(patchFlipVMInterfacesStates(vm)).To(Succeed())
+
+			vmi = waitForStatusToShowExpectedStates(vmName, map[string]v1.InterfaceState{
+				defaultInterfaceName: v1.InterfaceStateLinkDown,
+				bridgeInterfaceName2: v1.InterfaceStateLinkUp,
+			})
+			timeout = 30 * time.Second
+			Expect(console.RunCommand(vmi, assertLinkStateCmd(mac1.String(), v1.InterfaceStateLinkDown), timeout)).To(Succeed())
+			Expect(console.RunCommand(vmi, assertLinkStateCmd(mac2.String(), v1.InterfaceStateLinkUp), timeout)).To(Succeed())
+		})
+
+		It("hot plugging an interface with link state down and migrating vmi", func() {
```

Comment:
We have hotplug tests already and this feature does not deserve so much attention.
Please add it there, in one of the tests to see it works. You can check if there is a test with two interfaces and use it on one of them, or add such an interface to an existing test there.

Or, explain why you prefer a dedicated, yet another test, for hotplug + migration.